charts/incubator/alfresco-api-gateway/values.yaml

```diff
@@ -0,0 +1,17 @@
+
+registryPullSecrets: quay-registry-secret
```

Comment:
Do we need/want this if we're pulling from Docker Hub?

Reply:
Will remove.